website/content/en/docs/faqs/_index.md

```diff
@@ -238,3 +238,21 @@ allow configMaps larger than `~1MB`. However, from [OLM](https://github.com/oper
 
 The change to allow bigger bundles from [OLM](https://github.com/operator-framework/operator-lifecycle-manager) version `v0.19.0` only impacts the full bundle size amount. 
 Any single manifest within the bundle such as the CRD will still make the bundle uninstallable if it exceeds the default file size limit on clusters (`~1MB`).
+
+### The size of my Operator bundle is too big. How can I keep the CRDs smaller?
+
+We can design Operator solutions that have many APIs(CRDs). In this way, you might want to review the design of your Operator solution to ensure that it is not hurting concepts such as; encapsulation, the single responsibility principle, and cohesion. Also, you might need to find some alternative options to achieve the same goal in another way. To better understand the APIs, see [Groups and Versions and Kinds oh my!][kb-apis].
```

Comment:
this paragraph doesn't seem directly tied to keeping CRDs smaller, or even limiting the number of CRDs.  It seems like it could be dropped entirely.

Reply:
One of the reasons is to keep all specs in the same CRD instead of designing a solution that will have many APIs
So, here we are trying to make an overview that can be a reason but sometimes might be required to find some alternative solutions as well.

Reply:
I think we should never be encouraging a team to compromise their API design to fit in into an arbitrary OLM CSV size limit constraint.

Reply:
The idea here is clarified that a lack in the design might be a root cause for having CRDs too big.

Reply:
`but I think we can remove that.